fedmsg_meta_fedora_infrastructure/tests/pagure.py

```diff
@@ -1623,6 +1623,70 @@ class TestGitCommit(Base):
          'topic': 'io.pagure.prod.pagure.git.receive'
     }
 
+class TestIssueDrop(Base):
+    """ These messages are published when a ticket is closed against a
```

Comment:
Is it when they are closed?  Or when they are deleted? or..?  Can we explain more what "drop" means here?